docs/NativeComponentsIOS.md

```diff
@@ -299,38 +319,44 @@ RCT_EXPORT_VIEW_PROPERTY(onChange, RCTBubblingEventBlock)
     }
   });
 }
+@end
 ```
 
-You can see we're adding an event handler property to the view by subclassing `MKMapView`.  Then we're exposing the `onChange` event handler property and setting the manager as the delegate for every view that it vends. Finally, in the delegate method `-mapView:regionDidChangeAnimated:` the event handler block is called on the corresponding view with the region data.  Calling the `onChange` event handler block results in calling the same callback prop in JavaScript.  This callback is invoked with the raw event, which we typically process in the wrapper component to make a simpler API:
+In the delegate method `-mapView:regionDidChangeAnimated:` the event handler block is called on the corresponding view with the region data. Calling the `onRegionChange` event handler block results in calling the same callback prop in JavaScript. This callback is invoked with the raw event, which we typically process in the wrapper component to make a simpler API:
 
 ```javascript
 // MapView.js
 
 class MapView extends React.Component {
-  constructor(props) {
-    super(props)
-    this._onChange = this._onChange.bind(this);
-  }
-  _onChange(event: Event) {
+  _onRegionChange = (event) => {
```

Comment:
Removed typehint (since none of the other code is using it). And removed the ugly .bind(this). Not sure if this is okay in the sample code, please review.